New Troll Characters

ercarp opened the pull request:
<!--
A basic changelog, goes to patch notes of the next version, so should be as short and informative as possible.
-->
## Changelog:
- Added **[Kazra'jin](https://wowpedia.fandom.com/wiki/Kazra%27jin)**.
    - Changed Daakara's death date from 639 to 609.
    - Gave both Kazra'jin and Daakara `physique_good_3` at birth.
    - Made Kazra'jin the ruler of Zul'Aman after Daakara dies.
- Added **[Chieftain Zul'Marosh](https://wowpedia.fandom.com/wiki/Chieftain_Zul%27Marosh)** as a character.
    - Made Zul'Marosh the holder of Zeb'Watha between 595-609.
- Added **[Apari](https://wowpedia.fandom.com/wiki/Apari)**, Talanji's childhood friend.
    - Changed the death dates of Rastakhan (630 to 616) and Yazma (620 to 616) to be accurate to BfA timeline.
    - Made Talanji the ruler of Zandalar after Rastakhan's death (in case we ever do post-BfA bookmarks).
- Added **[Ruuzlu](https://wowpedia.fandom.com/wiki/Ruuzlu)** as a character.
- Added **[Jin'do the Hexxer](https://wowpedia.fandom.com/wiki/Jin%27do)** as a character
    - Added the nickname _'the Hexxer'_.
    - Changed Jammal'an's death date from 609 to 611.
        - Made Jammal'an Jin'do's vassal from 588 onwards.
        - Jammal'an is in the employ of Jin'do starting in 581.
    - Made Jin'do the leader of the Atal'ai Tribe from 581 until his death in 608.
    - Gave Jin'do the County of Temple of Atal'Hakkar.
- Added **[Sul the Sandcrawler](https://wowpedia.fandom.com/wiki/Sul_the_Sandcrawler)** as a character.
    - Added _'the Sandcrawler'_ as a nickname.
    - Changed Frost King Malakk's death and birth dates to make him younger than Sul, who is quite old compared to him.
- Added **[High Priestess Mar'li](https://wowpedia.fandom.com/wiki/High_Priestess_Mar%27li)**.
- Added **[General Jakra'zet](https://wowpedia.fandom.com/wiki/General_Jakra%27zet)** as a character.
  - He is in Zul's employ from 613-616 until he is promoted to General and takes residence in Rastakhan's court instead.
- Added **[Vilnak'dor](https://wowpedia.fandom.com/wiki/Vilnak'dor)** as a character.
  - He is in Rastakhan's employ from 598 until his death in 616.

## Tasks:
- [x] Add [Ruuzlu](https://wowpedia.fandom.com/wiki/Ruuzlu), friend and right-hand man of Chief Ukorz Sandscalp.
- [x] Add [Apari](https://wowpedia.fandom.com/wiki/Apari), daughter of Yazma and close friend of Princess Talanji.
- [x] Add [Jin'do the Hexxer](https://wowpedia.fandom.com/wiki/Jin%27do), the original leader of the Hakkari.
- [x] Add [Chieftain Zul'Marosh](https://wowpedia.fandom.com/wiki/Chieftain_Zul%27Marosh), ruler of Zeb'Watha.
  - [ ] (Optional) Correct the grammar of Zeb'watha to Zeb'Watha.
- [x] Add [Vilnak'dor](https://wowpedia.fandom.com/wiki/Vilnak'dor), former friend of Rastakhan.
- [x] Add [General Jakra'zet](https://wowpedia.fandom.com/wiki/General_Jakra%27zet), member of the Zanchuli Council and a central character during BfA.
- [x] Add missing members of the [Council of Elders](https://wowpedia.fandom.com/wiki/Council_of_Elders).
  - [x] Add [Kazra'jin](https://wowpedia.fandom.com/wiki/Kazra%27jin).
  - [x] Add [Sul the Sandcrawler](https://wowpedia.fandom.com/wiki/Sul_the_Sandcrawler).
  - [x] Add [High Priestess Mar'li](https://wowpedia.fandom.com/wiki/High_Priestess_Mar%27li).

<!--
Before the merge, we recommend you to do these tests with your branch.
-->
## Tests:
- [ ] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log`
- [ ] The mod takes less than 5.5 GB in the Task Manager (Windows)

<!--
If you need to explain something to testers. Otherwise, delete this part.
-->
# How to test:
- Provide feedback on the new characters added, especially in regards to their traits and stats, and make sure the additions don't cause any problems.

zumbak04 commented on history/characters/200000_farraki.txt:
```diff
@@ -8632,3 +8634,57 @@
 	718.9.14={ death=yes }
 }
+ruuzlu = { # right-hand man and friend of Ukorz Sandscalp
+	name = Ruuzlu
```
The name need localization.
> [16:15:34][characterhistory.cpp:698]: Missing loc for name 'Ruuzlu' for character 'ruuzlu'

ercarp replied:
Added in 193de1154b6fba4cf35e31de3e5d4fa26faa909a.

zumbak04 commented on history/characters/200000_farraki.txt:
```diff
@@ -8632,3 +8634,57 @@
+	martial=9 diplomacy=5 stewardship=5 intrigue=8 learning=5
+	trait=education_martial_4 # his strategic thinking earned him the position of General
+	trait=desert_warrior
+	trait=diligent trait=deceitful trait=zealous
```
Looks like his main drive was revenge on the Horde for the fall of his tribe.
Especially considering the fact he converted to the G'Huun heresy.
```suggestion
	trait=diligent trait=deceitful trait=vengeful
```

zumbak04 commented on history/characters/200000_farraki.txt:
```diff
@@ -8632,3 +8634,57 @@
+		employer = 66008 # Zul
+	}
+	613.12.10 = { # promoted to General of the Zandalari army after the Pandaria campaign
+		employer = 66000 # Rastakhan
```
So this will make him marshal. Do you want other characters to be councilors as well?
```suggestion
		employer = 66000 # Rastakhan
		give_council_position = councillor_marshal
```

ercarp replied:
> Do you want other characters to be councilors as well?

I think it would make sense if Vilnak'dor was the Marshal before Jakra'zet is promoted. I'll add it.

zumbak04 commented on history/characters/80000_drakkari.txt:
```diff
@@ -373,8 +373,12 @@
+	544.5.9={ birth=yes }
+	613.7.5={ # slain by adventurers in Throne of Thunder
+		death = {
+			death_reason = battle
```
```suggestion
			death_reason = death_battle
```

zumbak04 commented on history/titles/00_k_zandalar.txt:
```diff
@@ -9,6 +9,7 @@
 	299.6.5 = { holder = tezlekhan } # Tezlekhan[8330]
 	367.6.10 = { holder = aakul } # A'akul[8330]
 	462.11.11 = { holder = 66000 } # Rastakhan[8330]
+	616.12.11 = { holder = 66001 } # Talanji
```
Not all Rasta titles are transferred. Is it intended?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132106394-7598468f-39e8-4bbf-9a53-c6255ad91cf7.png)

</details>

ercarp replied:
Not intended. I thought I gave her all of Rastakhan's titles, but I guess not. I will fix it.

ercarp replied:
Fixed in 9e8a6cfd08a0171e80d38727749aeb463f4db0f5.

Remerod reviewed (changes requested):
Had a look and some suggestions for the Characters, I have only written traits and such since i don't think this was a pull to add feedback on how they looked? (because i had a few opinions on that to). 
<details>
<summary>Click to expand</summary>

**Daakara**
![image](https://user-images.githubusercontent.com/79544635/132109059-8e636a03-abd2-4828-8a8a-6207b1b31bd3.png)
> Daakara is the Warlord of the Amani tribe, successor to Zul'jin and the final boss of the Zul'Aman instance. With the aid of the Zandalari, the remaining Amani have reassembled under new, sturdy leadership in Daakara, seeking to avenge Zul'jin's death and realize his dreams of rebuilding the troll empire

Based on this iagree that Daakara have Ambitious, since he seeks to restore the amani empire. I'd possible give him the Vengeful trait becouse he wish to avange the death of Zul'jin but not as important.

**Kazra'jin**
![image](https://user-images.githubusercontent.com/79544635/132109106-1d3f8558-e775-41f2-bebe-4503b0b9b3c6.png)
Based on the text on how Kazra'jin throws his lot with the Zandalari and become one of the council of elders, I think trusting and ambitious are good traits as they are for now, possible other traits could be either the Gregorius or the Diplomat traits, but it's good for now.

**Chieftain Zul'Marosh** 
![image](https://user-images.githubusercontent.com/79544635/132109212-12f421f0-28f7-4efe-8db8-a685e19cf984.png)
In the quest Zul'Marosh the quest giver says;
>Come closer, mon. Don't be 'fraid, Ven'jashi not gonna hurt you.
Us Darkspears be mortal enemies of the Amani just like you. Their boss Zul'Marosh is evil troll. He give me the bad poison and leave me in this cage to die.
I not scared of dyin', mon. I seen death in the eye an' I laugh at her.
I cannot die in peace yet, mon. Not 'til Zul'Marosh be dead too. You find him 'cross the lake in Zeb'Watha. I hold this poison off 'til you bring me his big ugly head, mon. Only then I die peacefully. 

Having him with the Deceitful & Wrathful traits seems good to me, if i would add anything it would be the torturer trait based on how he treats his prisoner, and possible the Reaver trait because i guess he would like to raid the elves

**Apari**
![image](https://user-images.githubusercontent.com/79544635/132109380-115b5c5d-ea2d-4cad-9a01-719e61aedda5.png)
Based on the events later in life and how quickly she rebels, i'd give her the wrathful or Vengeful trait, witch you have. but i guess you could argue that she would not get those traits before she is left to die on the battlefied. So i say she looks good as now

**Ruuzlu**
![image](https://user-images.githubusercontent.com/79544635/132109487-b140c65f-c108-4337-b7ba-f097deec5fe0.png)
In the quest Wrath of the Sandfury it is written;
>De Sandfury are a resilient tribe. Dey have survived sandstorms an' years an' years of other tings long covered by da sands. It would be foolish to assume that the only forces here are da trolls. We been tamin' da scarabs, breedin' da sul'lithuz, an' raisin' our dead to keep on fightin' for all eternity as zombies. Ruuzlu, da right hand of Chief Ukorz Sandscalp, be plannin' to use dem to retake Tanaris for de Sandfury empire. You best be killin' da trolls in charge of dis if you want to live, mon. 

So i'd give him either the Strategist trait or the Schemer trait based on him having plans.
Also i lower his intrigue power from 23 to under 20 and heighten martial from 9 to something 10+ based on that his ingame class in wow is a warrior, so perhaps even his education should be more warrior based. But otherwise he looks good.

**Jin'do the Hexxer**
![image](https://user-images.githubusercontent.com/79544635/132109733-81273315-5262-4902-8a15-6deb812de8d8.png)
perhaps give him the Mystic trait? but i dunno, he looks good to me.

**Sul the Sandcrawler**
![image](https://user-images.githubusercontent.com/79544635/132109839-464d733a-8794-4f73-9ab3-67d50d389fc8.png)
I dunno, he kinda looks good. perhaps add the Desert Warrior or hunter trait to him, so have something to him?

**High Priestess Mar'li**
![image](https://user-images.githubusercontent.com/79544635/132109924-9d50f261-0b0d-454a-8949-1e8b23854462.png)
She looks okay for the most part... but since she is a high priest of Shadra latter on... perhaps she should have slightly more intrigue then 0? Because you know spiders, poison and schemes and such go hand in hand.
and when i say slightly I in reality mean a lot more intrigue.

**General Jakra'zet**
![image](https://user-images.githubusercontent.com/79544635/132110050-c5927dcf-2397-46ec-b525-9a98569f0455.png)
Looks good, if anything i would add the strategist trait. 

**Vilnak'dor**
![image](https://user-images.githubusercontent.com/79544635/132110110-c4e0b5d3-220d-4797-be1d-0ece7491fea9.png)
Looks good, put perhaps add something to make him a better in prowess & also he have skills in diplomacy with the mogu, so i think better learning and a diplomat trait should be added to represent this. and later he drain power from Rastakhan to give to Zul, and seems to have other abilities as well. 
![image](https://user-images.githubusercontent.com/79544635/132110136-59d58be2-fdf7-4453-8a53-dc40ae2ccc88.png)
So we should perhaps see him as some kind of warrior shaman? perhaps add mystic or something other to represent this? if i am not remembering wrong we might already develop shaman abilities and such but those might not be in yet? 

</details>

zumbak04 commented:
> She looks okay for the most part... but since she is a high priest of Shadra latter on... perhaps she should have slightly more intrigue then 0? Because you know spiders, poison and schemes and such go hand in hand.
and when i say slightly I in reality mean a lot more intrigue.

I would switch her education to intrigue then.

ercarp commented:
> **Daakara**
> Based on this iagree that Daakara have Ambitious, since he seeks to restore the amani empire. I'd possible give him the Vengeful trait becouse he wish to avange the death of Zul'jin but not as important.

Daakara wasn't among the new trolls I added, but I've gone ahead and replaced his **_Gregarious_** trait with **_Vengeful_**. It seemed appropriate.

> **High Priestess Mar'li**
> She looks okay for the most part... but since she is a high priest of Shadra latter on... perhaps she should have slightly more intrigue then 0? Because you know spiders, poison and schemes and such go hand in hand.
and when i say slightly I in reality mean a lot more intrigue.
>
> I would switch her education to intrigue then.

@Remerod @zumbak04 I gave her an Intrigue education and rebalanced her stats.

> **General Jakra'zet**
> Looks good, if anything i would add the strategist trait.

Added in 533fab29ce27b6d2ef824e3135101d478730c903.

> **Jin'do the Hexxer**
> perhaps give him the Mystic trait? 

Added in 533fab29ce27b6d2ef824e3135101d478730c903.

> **Zul'Marosh**
> if i would add anything it would be the torturer trait based on how he treats his prisoner

Added in 533fab29ce27b6d2ef824e3135101d478730c903.

> **Vilnak'dor**
> Looks good, put perhaps add something to make him a better in prowess & also he have skills in diplomacy with the mogu, so i think better learning and a diplomat trait should be added to represent this.
> perhaps add mystic or something other to represent this?

I gave him the **_Diplomat_** lifestyle trait, but I don't think having both Diplomat and Mystic would make sense from a gameplay perspective (although I'm not 100% sure how all of the lifestyle traits are acquired).

> **Sul the Sandcrawler**
> I dunno, he kinda looks good. perhaps add the Desert Warrior or hunter trait to him, so have something to him?

He's already very similar to Jakra'zet, so I don't think it's a good idea to also give both of them Desert Warrior.

> **Ruuzlu**
> So i'd give him either the Strategist trait or the Schemer trait based on him having plans.
Also i lower his intrigue power from 23 to under 20 and heighten martial from 9 to something 10+ based on that his ingame class in wow is a warrior, so perhaps even his education should be more warrior based. But otherwise he looks good.

I gave him a Martial education and rebalanced his stats.

> **Apari**
> Based on the events later in life and how quickly she rebels, i'd give her the wrathful or Vengeful trait, witch you have. **but i guess you could argue that she would not get those traits before she is left to die on the battlefied**. So i say she looks good as now

This is something I also considered, but I ended up deciding that she was always vengeful and it just wasn't apparent until that vengefulness was directed towards Talanji.

> **Kazra'jin**
> possible other traits could be either the Gregorius or the Diplomat traits, but it's good for now.

He is described as a very powerful troll so I thought it would make more sense if he was Martial-focused.

zumbak04 reviewed (approved):
LGTM!

Remerod reviewed (approved):
Look good to me to! I approve.

But, it would be nice if this optional thing you added on the list was fixed ;)
_(Optional) Correct the grammar of Zeb'watha to Zeb'Watha._
![image](https://user-images.githubusercontent.com/79544635/132382526-0cbf57b1-dcc3-4b1d-a417-b067b4151b7f.png)

ercarp commented:
> 
> 
> Look good to me to! I approve.
> 
> But, it would be nice if this optional thing you added on the list was fixed ;)
> _(Optional) Correct the grammar of Zeb'watha to Zeb'Watha._
> ![image](https://user-images.githubusercontent.com/79544635/132382526-0cbf57b1-dcc3-4b1d-a417-b067b4151b7f.png)

_I replied to you already on Discord but I'm pasting my reply here just so the discussion stays in the same place._

The reason I tagged it as "optional" is that I think the actual grammar and names for all of the provinces are defined through the MapFillers sheet, so it would get overwritten later when we add Broken Isles stuff. So I think it would be best to just remind @ElMariuso or someone to correct it to Zeb'Watha when he starts adding new provinces.
